src/pages/create/OfferForm.tsx

```diff
@@ -138,6 +140,13 @@ const OfferForm = () => {
     return undefined;
   }, [asPath, query.scope]);
 
+  const offerId =
+    scope === OfferTypes.EVENTS && typeof query.eventId === 'string'
+      ? query.eventId
+      : scope === OfferTypes.PLACES && typeof query.placeId === 'string'
+      ? query.placeId
+      : undefined;
```

Comment:
You could rewrite it to this:

```suggestion
 const offerId = arrayToValue(scope === OfferTypes.EVENTS ? query.eventId : query.placeId) || undefined
```

Reply:
Done ✅